README.md

```diff
@@ -1,2 +1,15 @@
-# drink-water
-# drink-water
+# Project Name
+
+Drink Water
+
+## Project Description
+
+This project calculates the amount of water drunk in a day
+
+## Instructions
```

Comment:
You can add two separate instructions as follows

1. Instruction on how to run the repo locally
2. Instruction for creating a PR